style.css

```diff
@@ -531,3 +531,34 @@ body > * > * {
   margin-top: -7px;
   border-width: 7px 7px 7px 0;
 }
+
+details {
```

Comment:
These styles are applied to all details elements on the website. Currently there are none, but we may way to use them later!

How about we use a class in place of the `details` element selector so that we can use `details` elements in other places?

Reply:
I tried keeping this specific since this is aligned with the collapse/expand (: